buildSrc/src/main/java/Libs.kt

```diff
@@ -0,0 +1,45 @@
+import org.gradle.api.Project
+
+object Libs {
+    object AndroidX {
+        const val AppCompat = "androidx.appcompat:appcompat:${Versions.ANDROID_APP_COMPAT}"
+        const val ConstraintLayout =
+            "androidx.constraintlayout:constraintlayout:${Versions.ANDROIDX_CONSTRAINT_LAYOUT}"
+        const val LifecycleExtensions =
+            "androidx.lifecycle:lifecycle-extensions:${Versions.ANDROIDX_LIFECYCLE}"
+        const val LifecycleViewModel =
+            "androidx.lifecycle:lifecycle-viewmodel:${Versions.ANDROIDX_LIFECYCLE}"
+        const val LifecycleViewModelKtx =
+            "androidx.lifecycle:lifecycle-viewmodel-ktx:${Versions.ANDROIDX_LIFECYCLE}"
+        const val LifecycleReactiveStreams =
+            "androidx.lifecycle:lifecycle-reactivestreams-ktx:${Versions.ANDROIDX_LIFECYCLE}"
+        const val Material = "com.google.android.material:material:${Versions.ANDROID_MATERIAL}"
+    }
+
+    const val Picasso = "com.squareup.picasso:picasso:${Versions.PICASSO}"
+
+    class Trikot(project: Project) {
```

Comment:
You could define an extension property on Project named Trikot, so that its usage is similar to any other object.

```
val Project. Trikot: Trikot 
    get() = Trikot(this)
```

This can be referenced as `Trikot.Foundation` (instead of `Trikot(project).Foundation`)

Reply:
damn that's clever!

Reply:
Couldn't get it to work though, project is not the receiver and that complicate things.

Reply:
That worked on trikot.viewmodel-declarative, weird 🤔